substrate/frame/scheduler/src/lib.rs

```diff
@@ -1137,18 +1304,29 @@ impl<T: Config> Pallet<T> {
 					}
 					let wake = now.saturating_add(period);
 					match Self::place_task(wake, task) {
-						Ok(_) => {},
+						Ok(new_address) => {
+							if let Some(RetryConfig { total_retries, remaining, period }) =
+								Retries::<T>::take((when, agenda_index))
+							{
+								Retries::<T>::insert(
+									new_address,
+									RetryConfig { total_retries, remaining, period },
```

Comment:
```suggestion
									RetryConfig { total_retries, total_retries, period },
```
it would be more clear what's happening